packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -566,7 +566,7 @@ class CupertinoSliverNavigationBar extends StatefulWidget {
     this.padding,
     this.transitionBetweenRoutes = true,
     this.heroTag = _defaultHeroTag,
-    this.stretch = true,
+    this.stretch = false,
```

Comment:
the documentation of the class may need updating as well (currently it says the stretch property "Defaults to `true`").

Reply:
I did here right https://github.com/flutter/flutter/pull/74669/files#diff-0f85f66e28e2ac684db897552ae5d0e9c4f7ddf326175e91f66beed3110900c2R666? Or was there somewhere else I missed?

Reply:
Yeah it's mentioned here too:

https://github.com/flutter/flutter/blob/c8a1ea07249aa5ca750a00e82b89b18fadee4013/packages/flutter/lib/src/cupertino/nav_bar.dart#L541-L550

Reply:
thanks!